### PR TITLE
mgr/dashboard: Cluster Creation Workflow followups

### PR DESCRIPTION
**Pull request by nizamial09**
1. Fix bug in the modal where going forward one step on the wizard and coming back opens up the add host modal.
2. Renamed Create Cluster to Expand Cluster as per the discussions
3. A skip confirmation modal to warn the user when he tries to skip the cluster creation
4. Adapted all the tests
5. Did some UI improvements like fixing and aligning the styles, colors..
new

**UI improvements**
new

**Skip Confirmation Modal**
skipmodal

Fixes: https://tracker.ceph.com/issues/51640
Signed-off-by: Nizamudeen A <nia@redhat.com>

## Checklist
- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [x] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by nizamial09** [comment]
```
Running:  orchestrator/06-cluster-welcome-page.e2e-spec.ts                                (1 of 1)


  Create cluster page
    ✓ should fail to create cluster (2663ms)
    ✓ should skip to dashboard landing page (1474ms)


  2 passing (4s)
```

**Review by epuertat — APPROVED** [review]

**Comment by nizamial09** [comment]
Removed the e2e tests since those are being covered separately in https://github.com/ceph/ceph/pull/41951

**Review by avanthakkar — APPROVED** [review]
LGTM!

**Review by Waadkh7 — APPROVED** [review]

File: src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster.component.html [hunk 1]
```diff
@@ -63,3 +63,11 @@
                     [name]="showCancelButtonLabel()"></cd-back-button>
   </div>
 </div>
+
+<ng-template #skipConfirmTpl>
+  <span i18n>You are about to skip the cluster expansion process.
+             You’ll need to <strong>navigate through the menu to add hosts and services</strong></span>
+  <br>
+  <br>
```
Comment (alfonsomthd) [inline src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster.component.html:71]: IMHO it's more sustainable to [set margin/padding/...](https://getbootstrap.com/docs/4.5/utilities/spacing/) than to add a chain of <br />.

Comment (nizamial09) [reply src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster.component.html]: Done, thanks

File: src/pybind/mgr/dashboard/frontend/src/app/shared/components/wizard/wizard.component.html [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
-                <span i18n>{{ stepsTitle[i] }}</span>
-              </a>
-            </li>
-          </ul>
-        </nav>
-      </div>
-    </ng-container>
-  </div>
+<div class="card-body">
+  <ng-container>
```
[the remark this answers is not in the document]
Comment (nizamial09) [reply src/pybind/mgr/dashboard/frontend/src/app/shared/components/wizard/wizard.component.html]: You are right. Removed it.

**Review by alfonsomthd — APPROVED** [review]

**Pull request merged**